pyvlx/api/frames/frame_status_request.py

```diff
@@ -14,17 +15,18 @@ class FrameStatusRequestRequest(FrameBase):
 
     PAYLOAD_LEN = 26
 
-    def __init__(self, session_id=None, node_ids=None):
+    def __init__(self, session_id: Optional[int] = None, node_ids: Optional[List[int]] = None):
         """Init Frame."""
         super().__init__(Command.GW_STATUS_REQUEST_REQ)
         self.session_id = session_id
-        self.node_ids = node_ids
+        self.node_ids = node_ids if node_ids is not None else []
```

Comment:
I would have preferred `self.node_ids = node_ids  or []` . 

What do you think, should we make `node_ids` a set?

Reply:
I'm generally fonder of the more verbose forms, mainly because the form I used is explicit about which values for `node_ids` results in the empty set v.s. the one you proposed which to my mind kinda hides things.